src/test/scala/org/kibanaLoadTest/helpers/CloudHttpClient.scala

```diff
@@ -13,7 +13,8 @@ import spray.json.lenses.JsonLenses._
 import spray.json.DefaultJsonProtocol._
```

Comment:
Curious, what's the underscore mean here?

Reply:
I've seen it mean, "ignore this"...like a placeholder of something you dont care about.

Reply:
Wild card: all of `spray.json.DefaultJsonProtocol` is imported